src/train.py

```diff
@@ -109,11 +107,13 @@ def modeling(
     return
 
 
-@hydra.main(config_name="runs/main", config_path="conf")  # TODO check how to use hydra without path override
+# TODO check how to use hydra without path override
+@hydra.main(config_name="runs/main", config_path="conf")
 def main(args):
 
     # Set logger
-    _logger = wandb if not args.experiment.debug else Mock()
+    # if not args.experiment.debug else Mock()
+    _logger = wandb
```

Comment:
why remove that and log debug runs?

Reply:
This was for my testing.
I will fix this in my next push :)